source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageRenderer.java

```diff
@@ -126,7 +136,20 @@ else if ((image = Image.dynamic_cast(baseCardElement)) == null)
 
         ImageView imageView = new ImageView(context);
         imageView.setTag(image);
-        ImageRendererImageLoaderAsync imageLoaderAsync = new ImageRendererImageLoaderAsync(renderedCard, imageView, hostConfig.getImageBaseUrl(), image.GetImageStyle());
+
+        String imageBackgroundColor = image.GetBackgroundColor();
+        int backgroundColor = -1;
+        if(!imageBackgroundColor.isEmpty())
+        {
+            backgroundColor = Color.parseColor(imageBackgroundColor);
```

Comment:
try & catch blocks?

Reply:
currently the UWP renderer drops the card if the color is not valid, will check with Gilles the fix for it

---
In reply to: [207017510](https://github.com/Microsoft/AdaptiveCards/pull/1749#discussion_r207017510) [](ancestors = 207017510)

Reply:
If i were you, if android renderer has infrastructures for warnings, i would use that; otherwise, i would just log it and put it on to-do list. 

---
In reply to: [207045249](https://github.com/Microsoft/AdaptiveCards/pull/1749#discussion_r207045249) [](ancestors = 207045249,207017510)